great-idea/css/index.css

```diff
@@ -62,5 +62,154 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+h3{
+	font-size: 25px;
+	margin-top: 20px;
+	
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+div{
+line-height: 1.25;
+}
+
+/* paragraphs */
```

Comment:
keep an eye on whitespace on the website, margins would have been a good way to accomplish this!